src/components/Icons.jsx

```diff
@@ -164,35 +164,55 @@ const Wrapper = WrappedComponent => props => {
     );
 };
 
+const  WrapIconWithSubtitle= WrappedComponent => props => {
+    let { className, ['subtitle']: subtitle, ...p } = props;
+    className = `ic-icon ic-icon-with-sub ${className ? className : ''} `;
+
+    return (
+        <span
+            className={className}
+            subtitle={subtitle}
+            {...p}>
+            <WrappedComponent />
+            <br/>
+            <span>{subtitle}</span>
+        </span>
+    );
+};
+
 export const AddIcon = Wrapper(Add);
+export const ZoomInIcon = WrapIconWithSubtitle(Add);
 export const AddThinIcon = Wrapper(AddThin);
-export const CandleIcon = Wrapper(Candle);
+export const CandleIcon = WrapIconWithSubtitle(Candle);
 export const CloseIcon = Wrapper(Close);
-export const ComparisonIcon = Wrapper(Comparison);
-export const CrosshairIcon = Wrapper(Crosshair);
+export const ComparisonIcon = WrapIconWithSubtitle(Comparison);
+export const CrosshairIcon = WrapIconWithSubtitle(Crosshair);
```

Comment:
I think creating `WrapIconWithSubtitle` is unnecessary complexity that limits the reusability of the icons. We don't need `tooltip-title` attribute anymore; a simpler solution is to place these tooltips in a `<span>` and selectively hide them when `<cq-context/>` has the class `smartcharts-mobile`.

Reply:
@brucebinary Yes you are right, it could be done by just style but since the icons are being used other palces too, they will still need the tooltip. right?

Reply:
@brucebinary  Yes you are right, it could be done by just styles but since the icons are being used other palces too like when you open Draw Tools there is Measure and Clear All buttons which need to remain as it is with tooltip and without subtitles. right?